ara/callback/log_ara.py

```diff
@@ -85,13 +85,28 @@ def __init__(self):
 
     def get_or_create_host(self, hostname):
         try:
-            host = models.Host.query.filter_by(name=hostname).one()
+            host = (models.Host.query
+                    .filter_by(name=hostname)
+                    .filter_by(playbook_id=self.playbook.id)
+                    .one())
         except models.NoResultFound:
-            host = models.Host(name=hostname)
+            host = models.Host(name=hostname, playbook=self.playbook)
             db.session.add(host)
 
         return host
 
+    def get_or_create_file(self, path):
+        try:
+            file_ = (models.File.query
```

Comment:
Because `file` is a built-in Python type (that's what, for example, the `open` command returns), and I try avoid masking those.